stdlib/effect.mli

```diff
@@ -16,10 +16,7 @@
 
     @since 5.0 *)
 
-type _ t = ..
```

Comment:
We could re-export the built-in type here: `type 'a t = 'a eff = ..`.

Reply:
(Apologies, I see this has already been brought up.)

Reply:
Done. :-)